src/App.js

```diff
@@ -1,13 +1,21 @@
+import React from "react";
+
 import Header from "./components/header";
 import Breadcrumb from "./components/breadcrumb";
+import SubHeader from "./components/subheader";
+import FilterSection from "./components/filterSection";
+
```

Comment:
You're ordering your imports that's correct, also try to give a comment at the top of one group of imports such as components or icons